keel-orca/src/main/kotlin/com/netflix/spinnaker/keel/orca/OrcaService.kt

```diff
@@ -31,28 +32,49 @@ interface OrcaService {
 
   @POST("/ops")
   @Headers("Content-Type: application/context+json", "X-SPINNAKER-USER-ORIGIN: keel")
-  suspend fun orchestrate(@Header("X-SPINNAKER-USER") user: String, @Body request: OrchestrationRequest):
-    TaskRefResponse
+  suspend fun orchestrate(
+    @Header("X-SPINNAKER-USER") user: String,
+//    @Header("X-SPINNAKER-ACCOUNTS") accounts: Set<String>,
+    @Body request: OrchestrationRequest
+  ): TaskRefResponse
 
   @POST("/orchestrate/{pipelineConfigId}")
   @Headers("Content-Type: application/context+json", "X-SPINNAKER-USER-ORIGIN: keel")
   suspend fun triggerPipeline(
     @Header("X-SPINNAKER-USER") user: String,
+//    @Header("X-SPINNAKER-ACCOUNTS") accounts: Set<String>,
     @Path("pipelineConfigId") pipelineConfigId: String,
     @Body trigger: HashMap<String, Any>
   ): TaskRefResponse
 
   @GET("/pipelines/{id}")
-  suspend fun getPipelineExecution(@Path("id") id: String): ExecutionDetailResponse
+  suspend fun getPipelineExecution(
+//    @Header("X-SPINNAKER-USER") user: String,
+//    @Header("X-SPINNAKER-ACCOUNTS") accounts: Set<String>,
```

Comment:
The goal was to avoid having to add these 2 headers to dozens of service calls...